dataikuapi/dss/ml.py

```diff
@@ -633,26 +634,30 @@ def compute_subpopulation_analyses(self, split_by, wait=True, sample_size=1000,
             )
             future = DSSFuture(self.saved_model.client, future_response.get("jobId", None), future_response)
         if wait:
-            return future.wait_for_result()
+            return DSSSubpopulationAnalyses(future.wait_for_result())
         else:
             return future
 
 
     def get_subpopulation_analyses(self):
         """
-        Retrieve all subpopulation analyses computed for this trained model as a dict
+        Retrieve all subpopulation analyses computed for this trained model
+        
+        :returns: the subpopulation analyses
+        :rtype: :class:`dataikuapi.dss.ml.DSSSubpopulationAnalyses`
         """
         
         if self.mltask is not None:
-            return self.mltask.client._perform_json(
+            data = self.mltask.client._perform_json(
                 "GET", "/projects/%s/models/lab/%s/%s/models/%s/subpopulation-analyses" %
                 (self.mltask.project_key, self.mltask.analysis_id, self.mltask.mltask_id, self.mltask_model_id)
             )
         else:
-            return self.saved_model.client._perform_json(
+            data = self.saved_model.client._perform_json(
                 "GET", "/projects/%s/savedmodels/%s/versions/%s/subpopulation-analyses" %
                 (self.saved_model.project_key, self.saved_model.sm_id, self.saved_model_version),
             )
+        return DSSSubpopulationAnalyses(data)
 
     def compute_partial_dependencies(self, features, wait=True, sample_size=1000, random_state=1337, n_jobs=1, debug_mode=False):
```

Comment:
- Auto-wrap `features` if a single string is provided, otherwise the error message is hard to understand (cannot parse the class from body)
- Ditto for `compute_subpopulation_analyses`, auto-wrap `split_by`